include/sch/S_Polyhedron/Polyhedron_algorithms.h

```diff
@@ -77,6 +100,9 @@ namespace sch
 
     SCH_API void openFromFile(const std::string& filename);
 
+    SCH_API void getEdges();
```

Comment:
It is strange to have a `getXXX` function returning nothing.
It would be useful to have a comment on what the function is doing.